site/content/docs/5.3/content/tables.md

```diff
@@ -96,27 +119,40 @@ Use .table-striped-columns to add zebra-striping to any table column.
 
 {{< table class="table table-striped-columns" caption="Boosted striped columns table" >}}
 
-These classes can also be added to table variants:
+These classes can also be added to table variants.
```

Comment:
You could directly use this sentence in the `<summary>` instead of "See Bootstrap's examples"